src/components/Kanban/Board.tsx

```diff
@@ -146,7 +146,7 @@ export function KanbanSection<T extends { id: string }>(
         <div
           ref={provided.innerRef}
           className={
-            "relative mr-2 w-[300px] shrink-0 rounded-xl bg-secondary-200"
+            "relative mr-2 w-[300px] shrink-0 rounded-xl bg-secondary-200 bshadow"
```

Comment:
🤔 


(1) Check the other examples of border usage elsewhere, use tailwind border instead of creating a custom class.
(2) Also, make sure to use names that make sense - why refer to this as shadow when it's a border class?

Reply:
@gurramkarthiknetha same here ☝️  Use shadow